src/main/java/com/simple/autocomplete/title/AutocompleteController.java

```diff
@@ -0,0 +1,105 @@
+package com.simple.autocomplete.title;
+
+import com.simple.autocomplete.title.domain.TitleInfo;
+import com.simple.autocomplete.title.service.AutoCompleteService;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Set;
+
+@Controller
+@RequestMapping("/")
```

Comment:
아래 내용을 보니 공통으로 `/autocomplete` 가 포함되어 있네요.
```suggestion
@RequestMapping("/autocomplete")
```

이렇게 하고 아래에서 그 뒤에 따른 path를 적어주는것도 좋아보입니다.